alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -289,13 +289,10 @@ public static function handleConvertLiveSegmentFinished(BatchJob $dbBatchJob, kC
 		if(count($files) > 1)
 		{
 			// find replacing entry id
-			
-			$replacingEntry = kFlowHelper::getReplacingEntry($recordedEntry, $asset);
-			if(is_null($replacingEntry)) {
-				KalturaLog::err('Failed to retrieve replacing entry');
-				return $dbBatchJob;
-			}
-			
+			$replacingEntry = self::getReplacingEntry($recordedEntry, $asset);
+			if(is_null($replacingEntry))
+				KalturaLog::err("Failed to get replacing entry");
+		
```

Comment:
If we continue we might have a fatal on line #304 since $replacingAsset will be NULL